packages/react-components/react-motion-components-preview/library/src/types.ts

```diff
@@ -68,4 +79,6 @@ export type BaseAtomParams = {
   duration: number;
   /** The easing curve for the motion. */
   easing?: string;
+  /** Time (ms) to delay the animation. */
+  delay?: number;
```

Comment:
Let's do something like this, since most of these are native keyframe config options

```ts
/**
 * Common parameters shared by all atom functions (without delay).
 */
export type BaseAtomParams = {
  /** The functional direction of the motion: 'enter' or 'exit'. */
  direction: PresenceDirection;
} & Pick<KeyframeAnimationOpeions, 'duration' | 'easing' | 'delay'>;
```

Reply:
Done.